src/docker-compose.yaml

```diff
@@ -1,34 +1,61 @@
-version: '3.8'
+version: "3.9"
 
 services:
-  backend:
-    build: ./planscape
-    image: planscape/backend
+  web:
+    build:
+      context: .
+      dockerfile: docker/Dockerfile
+    ports:
+      - "8000:8000"
+    env_file:
+      - .env
+    depends_on:
+      - db
+      - redis
+    volumes:
+      - /var/run/docker.sock:/var/run/docker.sock:ro
+      - .:/usr/src/app
+
+  redis:
+    image: redis:6.2-alpine
+    restart: always
+    ports:
+      - "6380:6379"
+    command: redis-server --save 20 1 --loglevel warning
     volumes:
-      - socket:/home/planscape/backend/socket
-    # Expose port 5432 to get access to the PostGIS database
-    expose:
-      - 5432
-    environment:
-      - PLANSCAPE_DATABASE_HOST=host.docker.internal
-      - PLANSCAPE_CORS_ALLOWED_HOSTS='planscape_nginx'
-      - PLANSCAPE_CORS_ALLOWED_ORIGINS='planscape_nginx'
+      - cache:/data/redis
+
+  celery_worker:
+    build:
+      context: .
+      dockerfile: docker/Dockerfile
+    command: celery -A app worker -E --loglevel INFO --concurrency 3
+    deploy:
+      replicas: 2
+    env_file:
+      - .env
     depends_on:
-      - frontend
+      - db
+      - redis
+    volumes:
+      - .:/usr/src/app
 
-  frontend:
-    build: ./interface
-    image: planscape/frontend
+  db:
+    image: postgis/postgis:15-3.4
+    # potentially more useful for Apple Silicon
+    # image: kartoza/postgis:15-3.4--v2023.08.31
+      POSTGRES_USER: postgres
+      POSTGRES_PASSWORD: postgres
+      POSTGRES_DB: postgres
+      POSTGRES_MULTIPLE_EXTENSIONS: postgis,hstore,postgis_topology,postgis_raster,pgrouting
     volumes:
-      - socket:/django_socket
+      - postgres:/data/postgres
     ports:
-      - 80:80
-
-  planscape_r:
-    build: ./planscape_r
-    image: planscape/backend_r
+      - 5433:5432
```

Comment:
funny, I guess I had this same port on my local docker-compose and didn't even question it. Otherwise I usually use a higher port like 25432 for Docker thingys.

It can be a little bit of a pain to reconfigure the MacOS postgres depending on how its installed, so maybe we leave it at 5433?